packages/testing/playwright/playwright-projects.ts

```diff
@@ -32,6 +32,10 @@ const CONTAINER_CONFIGS: Array<{ name: string; config: N8NConfig }> = [
 	{ name: 'multi-main', config: { queueMode: { mains: 2, workers: 1 } } },
 ];
 
+// Enable fullyParallel for standard Playwright runs (community PRs, local)
+// Disable for Currents orchestration
+const enableParallelism = !process.env.CURRENTS_RECORD_KEY;
```

Comment:
Can we do this on the CURRENTS_BATCH_SIZE instead? The coverage tests use a Currents Record Key but they run with 4 workers/parallel